.github/workflows/main.yml

```diff
@@ -1,20 +1,19 @@
 name: Build App
 
-on: 
+on:
   push:
-    branches: 
+    branches:
       - master
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     steps:
-    - name: Run a one-line script
-      uses: peter-evans/repository-dispatch@v1
-      with:
-        token: ${{ secrets.ACCESS_TOKEN }}
-        repository: paritytech/substrate.io
-        event-type: content-update
-        client-payload: '{"ref": "${{ github.ref }}", "sha": "${{ github.sha }}"}'
+      - name: Run a one-line script
+        uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{ secrets.ACCESS_TOKEN }}
+          repository: paritytech/substrate.io
+          event-type: content-update
+          client-payload: '{"ref": "${{ github.ref }}", "sha": "${{ github.sha }}"}'
```

Comment:
So if the content being checked doesn't conform to the style guidelines, then what happens? The merge fails? Or  CI makes a commit that fixes it? Or something else?

Reply:
See above; the only thing being changed in this PR is formatting.